pypeit/calibrations.py

```diff
@@ -598,6 +598,9 @@ def get_slits(self):
         # Check internals
         self._chk_set(['det', 'calib_ID', 'par'])
 
+        # Use slit-mask design if spectrograph is `keck_deimos`
+        maskdesign = True if self.spectrograph.spectrograph == 'keck_deimos' else False
```

Comment:
Lines like if spectrograph == keck_deimos are not really allowed in our coding model. This is why we have parsets and spectrograph files. My suggestion is to implement the a maskdesign parameter into the EdgeTrace parset and set this in the keck_deimos spectrograph file.

Reply:
Added maskdesign parameter into the EdgeTrace parset